lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -952,11 +970,16 @@ def setUp(self):
         self.student = UserFactory.create()
         CourseEnrollmentFactory(user=self.student, course_id=self.course.id)
 
+    @patch('lms.lib.comment_client.comment.Comment.thread')
```

Comment:
@jimabramson In this case, the patch IS necessary. The thread's commentable_id is needed at some point, but it is not yet set. I'm going to take another look, because it seems I address it in this manner here, while in that code snippet with the try/finally, I use a different method. I'm not sure which is better. I think I'll use the try method, since it's more explicit about what we're trying to accomplish.